pynamodb/tests/test_attributes.py

```diff
@@ -151,17 +151,31 @@ def test_utc_date_time_deserialize(self):
             attr.deserialize(tstamp.strftime(DATETIME_FORMAT)),
         )
 
-    def test_utc_date_time_deserialize_parse_args(self):
+    def test_dateutil_parser_fallback(self):
+        """
+        UTCDateTimeAttribute.deserialize
+        """
+        expected_value = datetime(2047, 1, 6, 8, 21, tzinfo=tzutc())
+        attr = UTCDateTimeAttribute()
+        self.assertEqual(
```

Comment:
@lita Agreed, but I would do that in a follow on PR.  Having two different testing styles in the same file would be a bit weird.